apps/fishing-map-e2e/src/e2e/map/analysis.cy.ts

```diff
@@ -0,0 +1,54 @@
+import { MAP_POPUP_EEZ_SECTION, SIDEBAR_TOGGLE_EEZ } from '../../constants/buttons'
+import { SEARCH_EEZ, SEARCH_EEZ_FULL_NAME } from '../../constants/search'
+import {
+  disablePopups,
+  getMapCanvas,
+  getSidebar,
+  scrollSidebar,
+  waitForMapLoadTiles,
+  waitForSidebarLoaded,
+} from '../../support/app.po'
+
+describe('See the creation of analysis for an area', () => {
+  before(() => {
+    cy.login(Cypress.env('apiAuthUser'), Cypress.env('apiAuthPass'))
+  })
+  beforeEach(() => {
+    disablePopups()
+    cy.visit('/')
+    waitForSidebarLoaded()
+    waitForMapLoadTiles()
+    scrollSidebar('bottom', 1000)
+    cy.getBySel(SIDEBAR_TOGGLE_EEZ).click()
+    cy.getBySel('map-search-button').click()
+    // I need to add a delay because it doesnt work propetly the autocomplete if we type so fast
+
+    cy.getBySel('map-search-input').click().type(SEARCH_EEZ, { delay: 200 })
+    cy.getBySel('map-search-results').findBySelLike('map-search-result').first().click()
+    getMapCanvas().click('center')
+    cy.getBySel(MAP_POPUP_EEZ_SECTION, { timeout: 10000 }).findBySelLike('open-analysis').click()
+  })
+
+  it('Should create an analysis for an EEZ area', () => {
+    getSidebar().findBySelLike('report-title').contains(SEARCH_EEZ_FULL_NAME)
+    cy.getBySel('source-tags').findBySelLike('source-tag-item').contains('AIS')
+
+    // Path tag is a needed element that should exist to draw the charts
+    cy.getBySel('report-activity-evolution').find('path').should('exist')
+    cy.getBySel('report-vessels-graph').find('path').should('exist')
+    cy.getBySel('report-vessels-table').findBySelLike('vessel').should('exist')
+  })
+
+  it('Should create an analysis for an EEZ area', () => {
```

Comment:
Why do we have two tests with the same name?  👉  https://github.com/GlobalFishingWatch/frontend/pull/2153/files#diff-303739775a7f1ddc13229986b1073f0dc8fb4c86924ea260a1fbd091d740c127R32